Frameworks/UIKit/StarboardXaml/CompositorInterface.mm

```diff
@@ -329,9 +329,10 @@ void SetParams(UIFont* font,
         _centerVertically = centerVertically;
         _lineHeight = [font ascender] - [font descender];
 
-        int mask = [font fontDescriptor].symbolicTraits;
-        _isBold = (mask & UIFontDescriptorTraitBold) > 0;
-        _isItalic = (mask & UIFontDescriptorTraitItalic) > 0;
+        _fontWeight = [font _fontWeight];
+        _fontStretch = [font _fontStretch];
+        _fontStyle = [font _fontStyle];
```

Comment:
all of the code you're changing here is gone in the CA refactor, but I suppose this can go in and I'll have to forklift it to our UILabel impl.  Seems straightforward enough, but please ping me when this goes in so I can do yet another merge :/